packages/scandipwa/src/util/Form/Extract.js

```diff
@@ -120,6 +132,19 @@ export const getDateTimeFormat = (type, dateFieldsOrder, timeFormat) => {
     return `${datePart } ${ timePart}`.trim();
 };
 
+/** @namespace Util/Form/Extract/adjustAmpmHours */
+export const adjustAmpmHours = (hours, ampm) => {
+    if (ampm === 'PM') {
+        return (+hours % HOURS_12H_COUNT) + HOURS_12H_COUNT;
```

Comment:
If you want to convert string to number please use `Number.parseInt` or `parseInt` function. Using `+value` to convert is kind of a hack and can be confusing.

Reply:
It is a matter of what approach it is more welcome on each particular project. 

If we take ScandiPWA, string to number conversion via `+` is used 20+ times.
If you search for `parseInt`, it is used in ScandiPWA only once.

So I'm just using the approach most often used on the project.

Plus, from my personal preference is to use the shorter `+`.

Reply:
Still, using `+` is a hack and not everyone is familiar with it.
If it was used before it does not mean it should be used everywhere.